lib/amqp.js

```diff
@@ -177,7 +177,8 @@ class Amqp {
             // pass to callback both decrypted content & original message
             callback(decryptedContent, message);
         } catch (err) {
-            log.error(err, 'Failed to process message #%j, reject', message.fields.deliveryTag);
+            log.trace(err, 'Failed to process message #%j, reject', message.fields.deliveryTag);
```

Comment:
Did you see sensitive data in `err`? If no, please, revert, we want to know the reason. If you do, change to `debug`, because it's quite useful information. The same in all other similar places.

Reply:
@PaulAnnekov please approve, that `err` doesn't contain potentially sensitive data, I will revert changes in this case.